src/PortAudio.jl

```diff
@@ -730,29 +730,35 @@ function PortAudioStream(
                     output_device.output_bounds.high_latency,
                 )
             end
-            if samplerate === nothing
-                samplerate = combine_default_sample_rates(
+            samplerate = if samplerate === nothing
```

Comment:
Interesting, I'd never seen this construct before but now I see that it works.
I probably would have used `isnothing` and the ternary operator `?` but it seems fine.

Reply:
Ok. I like doing it this way because, unlike the ternary operator, you can add multiple lines if you want.

Reply:
Just FYI it is possible to use multiple lines with ternary operators:
https://github.com/JeffFessler/MIRT.jl/blob/1be5a795a6ddb56f1478740a65033f9cf6caad55/src/fbp/sino_geom.jl#L543-L545